src/plugins/intel_cpu/src/emitters/plugin/x64/utils.hpp

```diff
@@ -88,6 +90,37 @@ class RegPrinter {
     static constexpr size_t reg_len = 8;
     static constexpr size_t reg_cnt = 16;
 };
+#endif
+
+// The class can be used for safe inernal call in JIT kernels:
+// make preamble (in constructor), postamble (in desctructor) and safe call with RSP aligning
+class JitSafeInternalCall {
```

Comment:
Do you think it is possible to add API for call args filling (smth like `add_parameter`)? It would encapsulate OS-specific logic inside (e.g. nuances of Windows/Linux ABIs), and make function calls composition even simpler.
Not sure that we should do it now, I am just interested in your opinion 😄